WorkloadManagementSystem/Client/Matcher.py

```diff
@@ -187,9 +187,25 @@ def _processResourceDescription( self, resourceDescription ):
         if resourceDescription.has_key( name ):
           resourceDict[name] = resourceDescription[name]
 
-      if resourceDescription.has_key( 'JobID' ):
+      if 'JobID' in resourceDescription:
         resourceDict['JobID'] = resourceDescription['JobID']
 
+      # Convert MaxRAM and NumberOfCores parameters into a list of tags
+      maxRAM = resourceDescription.get( 'MaxRAM' )
+      nCores = resourceDescription.get( 'NumberOfCores' )
```

Comment:
In https://github.com/DIRACGrid/DIRAC/pull/2673/files#diff-aba9526474f0d79f0a090e310b134bc3R428 these parameters are called 
```
`Processors_%s' % Processors 
'Memory_%s' % totalMemory
```